### PR TITLE
Update scdf common deps (see below)

### PR DESCRIPTION
**Pull request by onobc**
* Remove duplicated maven-source-plugin from `spring-cloud-dataflow-common/**`

* Update Awaitility and Semver libs

This commit removes the dependency management for `com.jayway.awaitility` in
favor of the Spring Boot managed version at the new `org.awaitility`
coordinates.

* Also, the Semver lib is updated to 0.10.2

### CODE REVIEW

File: spring-cloud-dataflow-common/pom.xml [hunk 1]
```diff
@@ -18,8 +18,7 @@
 	</parent>
 
 	<properties>
-		<jayway-awaitility.version>1.7.0</jayway-awaitility.version>
```
Comment (onobc) [inline spring-cloud-dataflow-common/pom.xml:21]: This managed by boot and its group has changed from `com.jayway.awaitility` to `org.awaitility`

File: spring-cloud-dataflow-common/pom.xml [hunk 2]
```diff
@@ -50,19 +44,6 @@
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
-	<build>
```
Comment (onobc) [inline spring-cloud-dataflow-common/pom.xml:53]: This is all specified in parent builds. I wanted to remove it here first in a small slice and if all goes well move to the rest of SCDF in a separate subsequent code proposal.

**Comment by cppwfs** [comment]
LGTM 
Squash and merge

**Comment by cppwfs** [comment]
Squashed and merged.   Thanks for the great contribution!

**Pull request closed without merging**